src/lib/expansion/expansion-panel.ts

```diff
@@ -137,6 +152,19 @@ export class MatExpansionPanel extends _MatExpansionPanelMixinBase
     return this.expanded ? 'expanded' : 'collapsed';
   }
 
+  ngAfterContentInit() {
+    if (this._lazyContent) {
+      // Render the content as soon as the panel becomes open.
+      this.opened.pipe(
+        startWith(null!),
```

Comment:
Why does this start with never?

Reply:
Because the `opened` stream won't emit if the panel is open by default. This ensures that we render any panels that are open on load.